app/controllers/client_products_controller.rb

```diff
@@ -0,0 +1,3 @@
+class ClientProductsController < ApplicationController
```

Comment:
Since there are no methods in this controller, this controller should probably be deleted. You could recreate it ff you end up creating ClientProduct methods, but as a rule you don't want any code in your projects that is not currently being used.